chart/kubeapps/templates/frontend/configmap.yaml

```diff
@@ -103,47 +103,6 @@ data:
       }
     {{- end }}
 
-      # Forward '/api/assetsvc' to '/assetsvc'
-      # but preserving the encoding (eg. '%2F' is not converted to '/')
-      # see https://serverfault.com/a/906479
-      # Ex: from "/api/assetsvc/what$2Fever?param=value"
-      #     it matches as $1="/what$2Fever" and $args="param=value"
-      #     downstream services will receive "/assetsvc/what$2Fever?param=value"
-      location ~* /api/assetsvc {
-        rewrite ^ $request_uri; # pass the encoded url downstream as is,
-        rewrite /api/assetsvc([^?]*) /assetsvc$1?$args break;
-
-    {{- if .Values.frontend.proxypassExtraSetHeader }}
-        proxy_set_header {{ .Values.frontend.proxypassExtraSetHeader }};
-    {{- end }}
-
-    {{- if .Values.frontend.proxypassAccessTokenAsBearer }}
-        # Google Kubernetes Engine requires the access_token as the Bearer when talking to the k8s api server.
-        proxy_set_header Authorization "Bearer $http_x_forwarded_access_token";
-    {{- end }}
-
-        proxy_pass {{ include "kubeapps.frontend-config.proxy_pass" . -}};
-      }
-
-      location ~* /api/kubeops {
-        # Keep the connection open with the API server even if idle (the default is 60 seconds)
-        # Setting it to 10 minutes which should be enough for our current use case of deploying/upgrading/deleting apps
-        proxy_read_timeout 10m;
-        rewrite /api/kubeops/(.*) /$1 break;
-        rewrite /api/kubeops / break;
-
-    {{- if .Values.frontend.proxypassExtraSetHeader }}
-        proxy_set_header {{ .Values.frontend.proxypassExtraSetHeader }};
-    {{- end }}
-
-    {{- if .Values.frontend.proxypassAccessTokenAsBearer }}
-        # Google Kubernetes Engine requires the access_token as the Bearer when talking to the k8s api server.
-        proxy_set_header Authorization "Bearer $http_x_forwarded_access_token";
-    {{- end }}
-
-        proxy_pass {{ include "kubeapps.frontend-config.proxy_pass" . -}};
-      }
-
       location ~* /apis {
```

Comment:
Well, this is also a reminiscence of our beloved Kubeops service, I guess, but it is mostly aimed at handling `can-i` and other auth stuff mainly, I think?

Reply:
Actually, I think it's the one further down (`/api/` not `/apis/`, the latter being kubeapps-apis)? But yes, you're right - that one has a proxy_pass of:
```
        proxy_pass {{ include "kubeapps.frontend-config.proxy_pass" . -}};
```
which I missed because I was grepping for kubeops in the config. But this helper does indeed just proxy to the kubeops service (not sure why it's labeled frontend-config).